docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -19,4 +19,130 @@ There you can find links to upgrade notes for other versions too.
 - follow instructions in [the separate article](upgrade-instructions-for-read-model-for-product-lists.md) to introduce read model for frontend product lists into your project ([#1018](https://github.com/shopsys/shopsys/pull/1018))
     - we recommend to read [Introduction to Read Model](/docs/model/introduction-to-read-model.md) article
 
+### Tools
+- we recommend upgrading PHPStan level to 4
```

Comment:
I have feeling that these instructions should be on a separate page (file). When you are reading it, you cannot distinguish, when it stops and another instruction starts...

Reply:
done